test/check-storage-cockpit

```diff
@@ -248,15 +273,38 @@ class TestStorageCockpitIntegration(VirtInstallMachineCase, StorageCase):
 
     @nondestructive
     def testStandardPartitionExt4(self):
+        """
+        Description:
+            Test using for the installation an ext4 partition created in Cockpit storage
+
+        Expected results:
+            - An ext4 partition can be created and used for the root mount point
+        """
         self._testStandardPartition("ext4")
 
     @nondestructive
     def testStandardPartitionXFS(self):
+        """
+        Description:
+            Test using for the installation an xfs partition created in Cockpit storage
```

Comment:
Could this description be reworded a bit? I get what it means, but it took me a while to figure out :-). What about e. g. "Test installation on an XFS partition created in Cockpit storage"?

Reply:
I have the same suggestion also for the btrfs scenarios below.